src/magic_folder/magicfolderdb.py

```diff
@@ -20,7 +21,8 @@
 )
 
 PathEntry = namedtuple('PathEntry', 'size mtime_ns ctime_ns version last_uploaded_uri '
-                                    'last_downloaded_uri last_downloaded_timestamp')
+                                    'last_downloaded_uri last_downloaded_timestamp '
+                                    'latest_snapshot')
 
```

Comment:
I think this echos another comment somewhat, but if we bundled "the database changes" separately I think using something besides `namedtuple` here would be good? (e.g. at `@attrs` class?). I would hope we don't need to keep many of these fields too, as they're pretty specific to the current way the up/downloader and conflict-detection works (`version`, anything with `_uri_` in it ...).

Also, some of these will be mirrored by the snapshot metadata, right? Like the timestamps?

Reply:
Thanks. I will read up on `@attrs` class.